flask_app/legacy/populate_db.py

```diff
@@ -406,16 +416,22 @@ def save_submission_to_database(conn, submission, data_template):
 
 #function that stripst columns where more than one value is allowed
 def stripping_method(cell):
-    if ',' in cell:
+    if isinstance(cell, float) and math.isnan(cell):
+        return []
```

Comment:
A missing value is encoded as "nan", which is parsed as a floating-point value. This is not ideal. In the long-term, we should parse the spreadsheet, rather than going through an intermediate yaml form, and we'll see if that's an issue in that case.